rails/benchmarks/activemodel_views.rb

```diff
@@ -0,0 +1,68 @@
+require 'bundler/setup'
+
+require 'json'
+require 'ffaker'
+
+require 'rails'
+require 'action_controller/railtie'
+require 'active_model'
+
+class HeavyController < ActionController::Base
+  def index
+    # @more_records = %w(one two three)
+  end
+end
+
+class HeavyView < ActionView::Base
+  # include ApplicationHelper
+  # include Rails.application.routes.url_helpers
```

Comment:
Do we need the lines that are commented out? Might be better to remove it if we don't need it in order to avoid any confusion.